README.md

```diff
@@ -26,15 +29,25 @@ Currently, JAX-RS HTTP method annotations are used for setting the HTTP method o
 ###Usage
 ApiFest Doclet requires the following environment variables:
 
+For both modes:
+
+- mode - the Doclet support two modes in the moment ("mapping" and "doc").They can be comma separated.
 - mapping.version - the version your API will be exposed externally;
+- application.path - the application path used to obtain all application resources, it will be preprended to each internal path;
+
+Only for the Doclet "mapping" mode:
+
 - mapping.filename - the name of the mapping configuration file that will be generated;
 - backend.host - the host(your API is running on) where requests should be translated to;
 - backend.port - the port of the backend.host;
-- application.path - the application path used to obtain all application resources, it will be preprended to each internal path;
 - defaultActionClass - the fully qualified action class that will be added if no action is declared in Javadoc annotations;
 - defaultFilterClass - the fully qualified filter class that will be added if no filter is declared in Javadoc annotations.
 
-If your project uses maven, here is an example integration of ApiFest Doclet in your pom.xml:
+Only for the Doclet "doc" mode:
+
+- mapping.docs.filename - the name of the mapping documentation file that will be generated.
```

Comment:
Please, add mapping.docs.filename as a parameter to the Doclet - additionalJOption in the example pom.xml below. Also, add an explanation that mapping.docs.filename "could be defined in your pom.xml file so they stick to the code" (same as for mapping.filename and mapping.version).